airbyte-integrations/connectors/source-okta/source_okta/config_migration.py

```diff
@@ -31,11 +31,12 @@ def should_migrate(cls, config: Mapping[str, Any]) -> bool:
             > False, otherwise.
             > Raises the Exception if the structure could not be migrated.
         """
-        return "domain" not in config
+        return "sub_domain" not in config and "domain" not in config
```

Comment:
This is murky territory. What if `domain` is in config, but `sudbomain` is not, and what if the domain setting is "myorg". Wouldn't you try using `something.myorg.com` as the API endpoint in that case unless it's migrated? 

My vote is to stick with how those names were previously used and add a new config input name. Naming will be wonky, but you avoid a breaking change that way.